.github/workflows/serverless-verify.yaml

```diff
@@ -33,17 +32,8 @@ jobs:
     if: github.event_name == 'pull_request'
     runs-on: ubuntu-latest
     steps:
-      - name: install libssh2
-        run: sudo apt update && sudo apt install libssh2-1-dev -y
-      - name: libgit2 installation
-        run: |
-          git clone https://github.com/libgit2/libgit2.git
-          cd libgit2 
-          git checkout v1.5.2
-          cmake . -DBUILD_TESTS=OFF -DBUILD_CLI=OFF -DUSE_SSH=ON
-          sudo make install
-          sudo ldconfig
       - uses: actions/checkout@v4
+      - uses: ./.github/actions/libgit2-installation
```

Comment:
I would suggest to rename the action to `install-libgit2` or `setup-libgit2`. It's more consistent with the `checkout` or `golangci-lint-action` or `setup-go`

Reply:
Thanks, I'll go with `setup-libgit2`